lib/features/Get Started now/view/GetStartedNow_screen.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:galaxyxgate/core/themes/app_colors.dart';
+import 'package:galaxyxgate/features/Get%20Started%20now/view/widgets/dark%20earth%20in%20space.dart';
+import 'package:galaxyxgate/features/Get%20Started%20now/view/widgets/get_start_now_screen_background.dart';
+import 'package:galaxyxgate/features/Get%20Started%20now/view/widgets/signup.dart';
+
+
+class GetStartNowScreen extends StatefulWidget {
```

Comment:
the file name is ( GetStartedNow_Screen.dart ) .. and that's not dart's convention in naming files.

it should be ( getStartedNow_screen.dart )

al folder/files naming must be like that, please check this for all folders and files.

and it's better to name it ( screen rather than view ).